### PR TITLE
Small change to pass null object for getUniqueId interface

### PR DESCRIPTION
**Pull request by ts-leojoseph-r**
# Description
For getUniqueId interface, we need to pass the null object in the request parameter section

# Checklist
- [ ] I wrote/updated tests for new/changed code
- [x] I ran `npm run build` without issues

### CODE REVIEW

File: js-miniapp-bridge/src/bridge.js [hunk 1]
```diff
@@ -93,6 +93,7 @@ MiniAppBridge.getUniqueId = function() {
   return new Promise(function(resolve, reject) {
     return MiniAppBridge.exec(
       "getUniqueId",
+      null,
```
Comment (ts-nitish-a-jain) [inline js-miniapp-bridge/src/bridge.js:96]: Should this be `null` or an empty placeholder object, being explicit in the usage?

Comment (ts-leojoseph-r) [reply js-miniapp-bridge/src/bridge.js]: I guess null is rational which is the representation of no value/request

**Comment by corycaywood** [comment]
Looks like this change was already included in the PR from @minh-rakuten, so maybe we can close this?

**Comment by ts-leojoseph-r** [comment]
Similar changes were pushed in another PR

**Pull request closed without merging**